workloads/kube-burner/common.sh

```diff
@@ -54,7 +54,14 @@ collect_pprof() {
 
 run_workload() {
   local CMD
-  curl -sS -L ${KUBE_BURNER_URL} | tar -xzC /tmp/ kube-burner
+  if [[ -n ${BUILD_FROM_REPO} ]]; then
+    git clone --depth=1 ${BUILD_FROM_REPO} kube-burner
```

Comment:
do we want to make a tempdir for this first to make sure we dont collide w/ anything that might currently be in the directory?

Reply:
fixed